tools/travis-upload-to-s3.sh

```diff
@@ -22,5 +22,6 @@ chmod +x tools/s3-parallel-put
 
 AWS_S3_HOST="${AWS_S3_HOST:-s3.amazonaws.com}"
 
-echo "Uploading"
+FILE_COUNT=$(find "${CT_REPORTS}" -type f | wc -l)
```

Comment:
How long does it take to calculate number of files to upload? Maybe it's not worth counting?

Reply:
very fast, less than a half of a second.

Reply:
We need to know number of files, because it would be a next optimization most likely.
Because each build generates 30k files (5k files per task * 6 tasks)

Maybe we can compress successful builds test results with zip before uploading...